src/mapDataManipulation/const.ts

```diff
@@ -1,6 +1,14 @@
+export type ColorRange = {
+  from: number;
+  to: number;
+};
```

Comment:
This type should probably be exported from the library? (index.js)

Reply:
True, and Effects type also. Will export it, thanks.

Reply:
Exported both types.